src/server/game/Entities/Object/Position.cpp

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>
```

Comment:
``` * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>, released under GNU GPL v2 license: https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-GPL2```

Reply:
This new file

Reply:
but if it's copied from TC?

Reply:
Not entirely, I made it myself, I used exactly the same file name, What do we do in that case?

Reply:
ok, then  we can make it AGPL :+1: